.eslintrc.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
         // e.g. "@typescript-eslint/explicit-function-return-type": "off",
         '@typescript-eslint/no-use-before-define': 'off',
-        'linebreak-style': ['warn', 'windows'],
+        'linebreak-style': ['warn', process.platform === 'win32' ? 'windows' : 'unix'], // https://stackoverflow.com/q/39114446/2771889
```

Comment:
not sure whether the build pipeline understands this

Reply:
We use it in EchoPedia and echo-search without any issue.

But I see some unit tests failed